libs/blocks/global-navigation/global-navigation.css

```diff
@@ -421,13 +426,19 @@ header.global-navigation {
   padding: 12px 0;
 }
 
-.feds-signIn-dropdown li > a {
+.feds-signIn-dropdown li > a,
+.feds-signIn-dropdown li > button {
```

Comment:
This will lead to issues as `a` has precedence, which is the reason I've added `li > a` in the first place 🤔 
Open for suggestions! 
![Screenshot 2023-09-01 at 12 09 35](https://github.com/adobecom/milo/assets/39759830/1c85a76c-0f3e-45c0-9421-d814f6c35334)